src/components/_pages/certificates/detail/index.tsx

```diff
@@ -88,6 +90,7 @@ export default function CertificateDetail() {
     const [raProfileOptions, setRaProfileOptions] = useState<{ label: string; value: string }[]>([]);
     const [userOptions, setUserOptions] = useState<{ label: string; value: string }[]>([]);
     const raProfileSelected = useSelector(raProfilesSelectors.raProfile);
+    const contentTypeEnum = useSelector(enumSelectors.platformEnum(PlatformEnum.CertificateRequestFormat));
```

Comment:
I guess instead of `contentTypeEnum` it should be `certificateRequestFormatEnum`